fv3config/config/diag_table.py

```diff
@@ -0,0 +1,212 @@
+import logging
+from typing import Sequence, Optional, Union
+import dataclasses
+import datetime
+import re
+
+import dacite
+
+from .._exceptions import ConfigError
+
+logger = logging.getLogger("fv3config")
+
+
+@dataclasses.dataclass
+class DiagTableField:
+    """Object representing a field of a diagnostics file."""
+
+    module_name: str
+    field_name: str
+    output_name: str
+    reduction_method: Optional[str] = "none"
```

Comment:
Do your really mean `Optional` here? In terms of typing this is roughly equivalent to `Union[str, None]`, which wouldn't make sense unless you intend for the user to provide something like `reduction_method: null` in a yaml file.

If you are just saying this option as a default, you don't need `Optional`.

```suggestion
    reduction_method: str = "none"
```

Reply:
Ah nice, thanks.